patches/tModLoader/Terraria/release_extras/start-tModLoaderServer.sh

```diff
@@ -1,23 +1,53 @@
 #!/usr/bin/env bash
-cd "$(dirname "$0")"
+cd "$(dirname "$0")" || exit
```

Comment:
why this?

Reply:
If for some reason it's unable to cd, it will exit rather than running the script in the current directory. Likely not needed, but it's safer.